utils/build.ps1

```diff
@@ -1782,7 +1775,7 @@ function Build-Foundation([Platform]$Platform, $Arch, [switch]$Test = $false) {
         -Arch $Arch `
         -Platform $Platform `
         -UseBuiltCompilers ASM,C,CXX,Swift `
-        -BuildTargets $Targets `
+        -SwiftSDK:$SDKRoot `
```

Comment:
Is this `-SwiftSDK` flag fixing something or was it optional?

Reply:
It is an optional flag; it seems that we cannot get away with it being set for Windows. I think that making this build always in that mode would be nice, but beyond the scope of the cleanup.